packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesUtils.kt

```diff
@@ -130,16 +128,15 @@ object UpdatesUtils {
       if (expectedBase64URLEncodedHash != null && expectedBase64URLEncodedHash != hashBase64String) {
         throw IOException("File download was successful but base64url-encoded SHA-256 did not match expected; expected: $expectedBase64URLEncodedHash; actual: $hashBase64String")
       }
-
       // only rename after the hash has been verified
       // Since renameTo() does not expose detailed errors, and can fail if source and destination
       // are not on the same mount point, we do a copyTo followed by delete
+      // if there are two assets with identical content, they will be written to the same file path,
+      // so we allow overwrites
       try {
-        tmpFile.copyTo(destination)
+        tmpFile.copyTo(destination, true)
```

Comment:
how would the finally section behave if two callback are trying to delete the file while one has overwritten the previous one?